scripts/deploy_testnet_instance.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+
+"""
+This script is a helper to deploy a Solido instance on the testnet.
+It will create and fund a maintainer account, a multisig owner account, and a multisig account.
+It will also deploy the Solido program, the Multisig program, and create a Solido instance.
+
+In order to run this script, you will need to have the Solana CLI and the Lido CLI installed.
+You will also need to have the Rust toolchain installed, and the Lido CLI must be compiled.
+See the Lido repository for more information.
+
+The script will populate the `.testnet-keys` directory with the generated keypairs
+if they are not already present there. It will also create a file named `solido_testnet_config.json`.
+
+The script will also need the SOLIDO_PROGRAM_PATH environment variable to point to the
+directory containing the compiled Solido program.
+
+The script will also need the NETWORK environment variable to be set to the network on which
+the Solido instance will be deployed. The default is the devnet. You can also set it to
+the testnet by setting the NETWORK environment variable to "https://api.testnet.solana.com".
+"""
+
+import os
+import json
+
+from util import (
+    TestAccount,
+    create_test_account,
+    run,
+    multisig,
+    maybe_from_file,
+    solido,
+    solana,
+    solana_program_deploy,
+    get_solido_program_path,
+    get_approve_and_execute,
+    get_network,
+    validators,
+    MAX_VALIDATION_COMMISSION_PERCENTAGE,
+)
+
+
+well_known_config_location = "solido_testnet_config.json"
+
+
+def main():
+    config = maybe_from_file(well_known_config_location)
+    if config is None:
+        config = {
+            "cluster": get_network() or "https://api.devnet.solana.com",
+        }
+    else:
+        config = json.loads(config)
+
+    # Hacky, but we don't have time to rework `utils` right now.
+    # Propagating `cluster` to the environment so that the whole `utils` see it.
+    os.environ["NETWORK"] = config["cluster"]
+
+    os.makedirs(".testnet-keys", exist_ok=True)
+
+    ### Checking the keys are present and creating them if not
+    maintainer = maybe_from_file(".testnet-keys/maintainer")
+    if maintainer is None:
+        print("\nGenerating maintainer keypair ...")
+        maintainer = create_test_account(".testnet-keys/maintainer")
+    else:
+        print("\nUsing existing maintainer keypair ...")
+        pubkey = run("solana-keygen", "pubkey", ".testnet-keys/maintainer").strip()
+        maintainer = TestAccount(pubkey, ".testnet-assets/maintainer")
+    print(f"> Maintainer is {maintainer}")
+
+    owner = maybe_from_file(".testnet-keys/owner")
+    if owner is None:
+        print("\nGenerating a multisig owner keypair ...")
+        owner = create_test_account(".testnet-keys/owner")
+    else:
+        print("\nUsing existing multisig owner keypair ...")
+        pubkey = run("solana-keygen", "pubkey", ".testnet-keys/owner").strip()
```

Comment:
We should generate new keys only in case, when we creating new instance.
If we trying to connect on existed instance, there are should be message "Please setup multisig_owner private key from secret."